### DISCUSSION TITLE
Source code Implementation documentation

### DISCUSSION THREAD

**Discussion by harhitosw** [tejado/Authorizer#71] [General]
I have been looking forward to contribute to Authorizer but I did not find any proper implementation details where I can understand whole flow of the app and what is happening through the code , I went through the readme.md file but I did not quite understand specifically what part of code implements what feature , can someone help me with this issue ?

**Comment by tejado** [comment]
Hi @harhitosw 
thats a really good point. I will create some documentation around this. Unfortunately - it doesn't exist so far.

**Reply by harhitosw** [reply]
Hello @tejado This is a great app that has many good features ! but for contributors to get some insights of current implementation we will require a proper documentation of what all we do and how do we do it ( flow of code ) . In terms of documenting you may add some comments in the code and provide a raw documentation of source code alongwith flow , rest part of refining it and adding more details I will do that from my end as I have a little experience in technical documentation .

**Reply by tejado** [reply]
Hey, I started to add some content in the Wiki: https://github.com/tejado/Authorizer/wiki/Dev-Overview
I will also add diagrams and co but this needs time which I then can't spend on the project. If you have specific questions, please let me know and I can document this.

**Reply by tejado** [reply]
@harhitosw any feedback? :) Added also a diagram.

**Reply by harhitosw** [reply]
Hello @tejado the diagram and crux points which you have wrote looks good to me . Being said this I am myself going through the code . Moreover if I find something which needs to be added we can discuss it here . Till than can you can update the readme.md accordingly . I will ping you here if I feel something :)
